content_fetch/fetch_gc_articles_pages.js

```diff
@@ -0,0 +1,59 @@
+const fetch = require('node-fetch');
+const buildFileName = require("../utils/buildFileName");
```

Comment:
Do you want to use the same shared import here?
```suggestion
const { fetch, buildFileName } = require('./shared');
```